src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,107 @@
+import { getNumbers } from '../../utils';
+
+interface Props {
+  items: string[];
+  itemsPerPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  itemsPerPage: ItemsPerPage,
+  currentPage,
+  onPageChange,
+}) => {
+  function creativePageContent(
+    data: string[],
+    listLength: number,
+    selectPage: number = 1,
+  ) {
+    const arrayOfSliceData = [];
+    const index = selectPage - 1;
+
+    for (let i = 0; i < data.length; i += listLength) {
+      arrayOfSliceData.push(data.slice(i, i + listLength));
+    }
+
+    if (arrayOfSliceData[index]) {
+      const pageContent = arrayOfSliceData[index];
+
+      return pageContent;
+    }
+
+    return 0;
```

Comment:
Returning `0` from the `creativePageContent` function when no page content is found is not appropriate, as it should return an empty array instead. This will prevent potential issues when mapping over the `content` variable later in the code.